### DISCUSSION TITLE
A chat-bot in Grafana UI

### DISCUSSION THREAD

**Discussion by dheeruymv** [grafana/grafana#49094] [Plugins]
Hello Team,

A simple chat-bot in "GRAFANA UI" would help many user's to know the things automatically via the bot.

Feature benefits:

- Reduce the user's time to search for some options in the user-guide or google.
- Bot can automate few tasks like "Creating a data-source", "Creating a user" etc. (This will reduce the time and effort for user's to manual navigate and create them)

and many more.

May be this can be added as plug-in as well!! Currently placing it under "General", based on feed-back we can tag to "Plug-ins"

If this feature is okay and approved, I would like to drive it with the help of everyone.

Thanks.

**Comment by dheeruymv** [comment]
Isn't this in the correct segment? No comments or any opinions?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Reply by SirArnoldB** [reply]
@dheeruymv, curious if there was there ever any separate follow up on this?

**Reply by gelicia** [reply]
> @dheeruymv, curious if there was there ever any separate follow up on this?

Please check out the announcement for Grafana assistant! I believe that is the desired implementation of this https://grafana.com/blog/2025/05/07/llm-grafana-assistant/

**Reply by SirArnoldB** [reply]
awesome, thanks!
will try this out.
